### DISCUSSION TITLE
File viewer not refreshing after updating file

### DISCUSSION THREAD

**Discussion by boshek** [posit-dev/positron#5548] [Q&A]
Starting the discussion here to ascertain if this is a bug or a skill issue. 

Using latest positron: 
```
Positron Version: 2024.12.0 (Universal) build 80
Code - OSS Version: 1.93.0
Commit: c3c489641477362638b2a80d94355a86a63eeb21
Date: 2024-11-26T02:46:47.938Z
Electron: 30.4.0
Chromium: 124.0.6367.243
Node.js: 20.15.1
V8: 12.4.254.20-electron.0
OS: Darwin arm64 23.6.0
```

Using R 4.4.1 

## Issue
- Open positron
- Create a parquet file `library(arrow); write_parquet(iris, "iris.parquet")`
- Open `iris.parquet` in viewer ✅ 
- Overwrite a parquet file `write_parquet(iris[,2:4], "iris.parquet")`
- Open `iris.parquet` in viewer ❌  <- does not update
- Confirm correct parquet actually got written `names(read_parquet("iris.parquet"))` ✅ 

So it seems like somewhere the parquet viewer in not updating. Also this is not an R thing. Also replicated with pandas. 

*Update* - replicated with a csv file as well
```r
write.csv(iris, "iris.csv")
write.csv(iris[,2:4], "iris.csv")
names(read.csv("iris.csv"))
```

**Comment by juliasilge** [comment]
Yep, I don't believe we have this kind of updating wired up for the headless/DuckDB Data Explorer, as opposed to the runtime (R and Python) based Data Explorer implementations. I have opened https://github.com/posit-dev/positron/issues/5551 to track and prioritize this feature. Thank you! 🙌
